src/Common/src/System/Net/Logging/NetEventSource.Common.cs

```diff
@@ -484,7 +485,7 @@ private static string Format(FormattableString s)
                 case 2: return string.Format(s.Format, Format(s.GetArgument(0)), Format(s.GetArgument(1)));
                 case 3: return string.Format(s.Format, Format(s.GetArgument(0)), Format(s.GetArgument(1)), Format(s.GetArgument(2)));
                 default:
-                    object[] args = s.GetArguments();
+                    object[] args = s.GetArguments()!;
```

Comment:
Why did you use ` !` here?  The array can contain nulls.  The type here should be `object?[]`.